flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -154,6 +203,11 @@ class FlxGamepad implements IFlxDestroyable
 		return gamepadButton;
 	}
 	
+	public inline function getFlipAxis(AxisID:Int):Int
```

Comment:
I originally added it to fix a bug that has since been fixed by other means, but in short, YES. The idea was to move as much weird hardware-and-platform specific fixup code and mapping logic to the SomeControllerID.hx classes and out of the FlxGamepadMapping class.